client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,673 @@
+	Solutions: i18n.translate( 'Solutions' ),
+	Songs: i18n.translate( 'Songs' ),
+	Space: i18n.translate( 'Space' ),
+	'Speed dating': i18n.translate( 'Speed dating' ),
+	Spiritual: i18n.translate( 'Spiritual' ),
+	'Spiritual coaching': i18n.translate( 'Spiritual coaching' ),
+	Spirituality: i18n.translate( 'Spirituality' ),
+	Sport: i18n.translate( 'Sport' ),
+	Sports: i18n.translate( 'Sports' ),
+	'Stationary supplies': i18n.translate( 'Stationary supplies' ),
+	Statistics: i18n.translate( 'Statistics' ),
+	'Steel protection': i18n.translate( 'Steel protection' ),
+	'Stock market': i18n.translate( 'Stock market' ),
+	Stocks: i18n.translate( 'Stocks' ),
+	Store: i18n.translate( 'Store' ),
+	Storefront: i18n.translate( 'Storefront' ),
+	Stories: i18n.translate( 'Stories' ),
+	Student: i18n.translate( 'Student' ),
+	'Student association': i18n.translate( 'Student association' ),
+	'Student housing': i18n.translate( 'Student housing' ),
+	'Students union': i18n.translate( 'Students union' ),
+	Style: i18n.translate( 'Style' ),
+	Stylist: i18n.translate( 'Stylist' ),
+	Subscriptions: i18n.translate( 'Subscriptions' ),
+	'Suicide prevention': i18n.translate( 'Suicide prevention' ),
+	Supplements: i18n.translate( 'Supplements' ),
+	Supplies: i18n.translate( 'Supplies' ),
+	Support: i18n.translate( 'Support' ),
+	'Sustainable development': i18n.translate( 'Sustainable development' ),
+	'Swimming pools': i18n.translate( 'Swimming pools' ),
+	'Swimming pools repairs': i18n.translate( 'Swimming pools repairs' ),
+	TV: i18n.translate( 'TV' ),
+	'TV shows': i18n.translate( 'TV shows' ),
+	'Tactical gear': i18n.translate( 'Tactical gear' ),
+	'Tattoo parlor': i18n.translate( 'Tattoo parlor' ),
+	Tattoos: i18n.translate( 'Tattoos' ),
+	Taxes: i18n.translate( 'Taxes' ),
+	'Taxi business': i18n.translate( 'Taxi business' ),
+	Taxidermy: i18n.translate( 'Taxidermy' ),
+	Teaching: i18n.translate( 'Teaching' ),
+	Tech: i18n.translate( 'Tech' ),
+	'Tech blog': i18n.translate( 'Tech blog' ),
+	'Technical recruiting': i18n.translate( 'Technical recruiting' ),
+	'Technical support': i18n.translate( 'Technical support' ),
+	Technology: i18n.translate( 'Technology' ),
+	Telecommunications: i18n.translate( 'Telecommunications' ),
+	Therapy: i18n.translate( 'Therapy' ),
+	Thoughts: i18n.translate( 'Thoughts' ),
+	Time: i18n.translate( 'Time' ),
+	Tips: i18n.translate( 'Tips' ),
+	Tourism: i18n.translate( 'Tourism' ),
+	'Tourism accommodation': i18n.translate( 'Tourism accommodation' ),
+	Toys: i18n.translate( 'Toys' ),
+	Trading: i18n.translate( 'Trading' ),
+	'Translation services': i18n.translate( 'Translation services' ),
+	Transport: i18n.translate( 'Transport' ),
+	Travel: i18n.translate( 'Travel' ),
+	'Travel agent': i18n.translate( 'Travel agent' ),
+	'Traveler blog': i18n.translate( 'Traveler blog' ),
+	Troubleshooting: i18n.translate( 'Troubleshooting' ),
+	Tutorials: i18n.translate( 'Tutorials' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Genesis Tutorials' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/translate-suggestions-about)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).